modules/persistent-storage-csi-efs-metrics-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-overview_{context}"]
+= Usage metrics overview
+
+Amazon Web Services (AWS) Elastic File Service (EFS) usage metrics allows you to monitor how much EFS space is used by {product-title}.
+
+[IMPORTANT]
+====
+This features is disabled by default, since turning on metrics can lead to performance degradation.
+====
+
+The AWS EFS usage metrics feature collects volume metrics in the AWS EFS CSI Driver by recursively walking through the files in the volume. Because this effort can degrade performance, administrators must explicitly enable this feature.
```

Comment:
Given the note above this section, don't think we need to duplicate this information? Or maybe put this paragraph as a note or combine this info?

Reply:
Just to be clear. 
```suggestion
The AWS EFS usage metrics feature collects volume metrics in the AWS EFS CSI Driver by recursively walking through the files in the volume. Because this effort can degrade performance, administrators must explicitly enable this feature, as shown in the following sections.
```